src/DOM/DOM.js

```diff
@@ -150,23 +154,25 @@ function DOMProperties (element, name, value, xmlns) {
 				return DOMStyle(element, value)
 			break
 		case 'className':
-			if (xmlns || value == null)
+			if (xmlns)
 				return DOMProperties(element, 'class', value, xmlns)
+			if (value === false)
+				break
 		case 'id':
 			return DOMProperty(element, name, value)
 		case 'width':
 		case 'height':
 			if (element.type === 'img')
-				break			
+				break
 		default:
 			if (!xmlns && name in DOMNode(element))
 				return DOMProperty(element, name, value)
 	}
 
-	if (value == null)
-		DOMProperties(element, name, false, xmlns)
+	if (typeof value === 'object')
```

Comment:
breaks `undefined`

Reply:
What do you mean by breaks `undefined`?

Reply:
https://github.com/thysultan/dio.js/issues/26